scip-java/src/main/scala/com/sourcegraph/scip_java/commands/IndexDependencyCommand.scala

```diff
@@ -70,12 +71,16 @@ final case class IndexDependencyCommand(
                       snapshotCommand
                         .copy(
                           output = snapshotTarget,
+                          cleanup = cleanup,
+                          targetroot = List(indexTarget),
```

Comment:
This line fixed the issue. The other diffs are not necessary but make the `--cleanup` flag work correctly, which is helpful for debugging in cases like this.